src/coord/src/coord.rs

```diff
@@ -3093,7 +3098,13 @@ impl Coordinator {
                     .collect();
                 // Persistence of system table inserts is best effort, so throw
                 // away the response and ignore any errors.
-                persist.write_handle.write(&updates, CmdResponse::Ignore);
+                //
+                // NB: Keep this method call outside the tokio::spawn. We're
```

Comment:
this is minor but if persistence of table is best effort (as per the comment above) then does it matter that this write happens in order?

Reply:
It might get reordering after a seal that turns it into an error. I'd call that not "best" effort, but ultimately no it doesn't really matter here. Added a note to the comment.